### DISCUSSION TITLE
Help hiding variants and not loosing association data

### DISCUSSION THREAD

**Discussion by jonathangkoehn** [rubiot/ibiblia#103] [Q&A]
I've gone a little ways in this project and realize I wish I had hid the variants from BYZa since I don't reference them much in Spanish. Though I guess I could translate them. (Will the verse rules come in?)

Here is an example.

![image](https://github.com/rubiot/ibiblia/assets/68037371/424f4dc1-3557-455a-972c-cdf324342514)

**Comment by rubiot** [comment]
You can replace the source text with a modified version of it, with the variants stripped out. Go to Project->Settings-Source tab->"Load new text" and select the new file. Click Yes when iBiblia ask you if you want to try to keep the existing associations. This generally works well, but please make a backup of your project before doing that.

If for some reason you don't have the original ByzA file, or don't want to deal with verse rules again, you can right-click an empty area of the source text and choose "Save to text file...", change it and reload it.

**Comment by rubiot** [comment]
Another point is, if you don't intend to export interlinear modules you can simply ignore the variants in the association process, if they don't bother you too much. But even if you want to export interlinears or want to see a clean interlinear in the chapter view, you can add verse rules to hide them. There's a context menu for the chapter view interlinear tabs that allow you to add verse rules and choose which ones are for displaying only and/or if they should also be exported along with the interlinear modules.
![image](https://github.com/rubiot/ibiblia/assets/8127450/d8d2ba3b-0d5c-4282-b1c6-41f7ba8657e6)
